spec/service_spec.rb

```diff
@@ -206,6 +206,30 @@
     end
   end
 
+  context "#api_uri" do
+    BASE_URI = "https://nobody.com"
+    API_PATH = "/ovirt-engine/api"
+
+    it "removes slash" do
+      allow(service).to receive(:base_uri).and_return(BASE_URI)
```

Comment:
You can share similar test setup in a `before` block in this context.  I'd prefer the `allow` to be an `expect` since we know that it will be called and should only be called once.